testdata/fsw-interface.jsonnet

```diff
@@ -0,0 +1,880 @@
+{
```

Comment:
Can you add a comment for what URL this was fetched from? That makes it easy for somebody else to update this.

Thanks!

Reply:
I added it, thanks